Makefile

```diff
@@ -8,9 +8,15 @@ build:
 clean:
 	docker-compose down -v --remove-orphans
 
+debug:
+	docker-compose kill app ; docker-compose -f docker-compose.debug.yml -f docker-compose.yml run --service-ports app
+
 requirements:
 	docker-compose run --rm -T app pip --quiet freeze > requirements-freeze.txt
 
+restart:
+	docker-compose exec app pkill -f gunicorn ; docker-compose up -d -- app
```

Comment:
What's wrong with `docker-compose restart` or just ctrl+c?

Reply:
Yes, a docker-compose restart would work here.  It would be nice to be able to exec a gunicorn restart rather than restarting the container, but the way that this is set up that doesn't work.

Reply:
Moved to restart functionality